b_cfn_sagemaker_endpoint/refresh/function.py

```diff
@@ -0,0 +1,60 @@
+import os
+
+from aws_cdk.aws_iam import PolicyStatement, Effect
+from aws_cdk.aws_lambda import Function, Runtime, Code
+from aws_cdk.aws_sagemaker import CfnEndpoint, CfnEndpointConfig
+from aws_cdk.core import Construct, Stack, Duration
+
+
+class RefreshFunction(Function):
+    from . import source
+    SOURCE_PATH = os.path.dirname(source.__file__)
+
+    def __init__(
+            self,
+            scope: Construct,
+            id: str,
+            endpoint: CfnEndpoint,
+            endpoint_config_a: CfnEndpointConfig,
+            endpoint_config_b: CfnEndpointConfig,
+            wait_time: float
+    ):
```

Comment:
Missing detailed constructor docstring. It is extremely important what A and B means.

Reply:
done